Gems/EditorPythonBindings/Code/Source/PythonProxyBus.cpp

```diff
@@ -394,7 +394,7 @@ namespace EditorPythonBindings
 
                     // log the bus symbol
                     AZStd::string subModuleName = pybind11::cast<AZStd::string>(thisBusModule.attr("__name__"));
-                    PythonSymbolEventBus::Broadcast(&PythonSymbolEventBus::Events::LogBus, subModuleName, ebusName, behaviorEBus);
+                    PythonSymbolEventBus::QueueBroadcast(&PythonSymbolEventBus::Events::LogBus, subModuleName, ebusName, behaviorEBus);
```

Comment:
This QueueBroadcast calls should perform an `AZStd::move` on the string parameters to prevent unneccesary copies from being created.

For example the `subModuleName` variable here is about to go out of scope, therefore the queued function can steal it's contents.